gbpservice/contrib/nfp/tools/image_builder/nfp-pecan

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+#
+PID_FILE=/var/run/nfp_pecan.pid
+
+start_nfp_pecan () {
+    exec start-stop-daemon --quiet --start -m --pidfile $PID_FILE --background --chdir /usr/local/lib/python2.7/dist-packages/gbpservice/nfp/pecan/api --exec /usr/local/bin/pecan  configurator_decider config.py --  --mode  advanced
+}
```

Comment:
No.
configurator docker is ubuntu image.Since Ubuntu doesn't use systemd.
systemctl is only for RHEL.